### PR TITLE
Add binary support

### LINKED ISSUE #36: Return binary
In your comments you mention `it's not possible to send binary via AWS API Gateway as it expects JSON response from Lambda`. You can do it with:

serverless.yml
```yml
plugins:
  - serverless-apigwy-binary # <-- Add
...
functions:
  run:
    description: Run headless Chromium
    memorySize: 1536
    timeout: 30
    handler: handler.run

    events:
      - http:
          path: chrome
          method: get
          contentHandling: CONVERT_TO_BINARY. # <-- Add
```

handler:
```js
{
                        statusCode: 200,
                        isBase64Encoded: true,  // <-- Add
                        body: screenshot.toString('base64')
}
```

I'm working on a PR right now.

### PR DESCRIPTION
**Pull request by willfarrell**
Closes #36 

Includes
- refactored out HTML handlers to own files.
- added binary handlers
- Updated `serverless.yml` to support binary

Things I noticed:
- `import config from '../config'` doesn't exist
- `import { log, sleep } from '../utils'` -> `import { log, sleep } from './utils'`
- webpacks removed - unable to compile down for deployment.

TODO:
- [x] Add web pack back into develop
- [ ] Test binary endpoints

### CODE REVIEW

**Comment by willfarrell** [comment]
Thanks for the detailed feedback!
- ` .babelrc` removed - I was actually just thinking about making this change after I committed.
- `babel-preset-env` done. `transform-object-rest-spread` was still needed though.
- `config.js` removed.

- Yeah I left the `html` versions in for backwards compatibility. In case people still want them. We can drop them later.
- Yeah, I've been thinking about rewriting them to use promises, but didn't want to step on any toes. I don't see AWS supporting node v8 any time soon (1y min). 90% of my time on this PR has been spent on debugging webpack. I'll leave this decision up to you.

- Yeah ... including `serverless-apigwy-binary` is a simple as it's going to get. Let them maintain that part, cause I don't want too.

File: examples/serverless-framework/aws/serverless.yml [hunk 1]
```diff
@@ -10,21 +10,19 @@ provider:
...
-package:
-  exclude:
-    - ./**
-  include:
-    - ./src/**
-    - node_modules/chrome-remote-interface/**
-    - node_modules/commander/**
-    - node_modules/ws/**
-    - node_modules/ultron/**
+#package:
```
Comment (willfarrell) [inline examples/serverless-framework/aws/serverless.yml]: removed entirely

**Comment by willfarrell** [comment]
I refactored captureScreenshot to not require webpack. I ran into this while testing:
```bash
2017-07-10 17:20:29.804 (-06:00)	5a96035f-65c6-11e7-9516-0fad8ff6c4ce	@serverless-chrome/lambda Spawning headless shell
2017-07-10 17:20:29.815 (-06:00)	5a96035f-65c6-11e7-9516-0fad8ff6c4ce	@serverless-chrome/lambda ChromeLauncher No debugging port found on port 9222, launching a new Chrome.
2017-07-10 17:20:29.840 (-06:00)	5a96035f-65c6-11e7-9516-0fad8ff6c4ce	@serverless-chrome/lambda Launcher Chrome running with pid 12 on port 9222.
2017-07-10 17:20:29.841 (-06:00)	5a96035f-65c6-11e7-9516-0fad8ff6c4ce	@serverless-chrome/lambda Waiting for Chrome 0
2017-07-10 17:20:30.345 (-06:00)	5a96035f-65c6-11e7-9516-0fad8ff6c4ce	@serverless-chrome/lambda Waiting for Chrome 1
2017-07-10 17:20:30.346 (-06:00)	5a96035f-65c6-11e7-9516-0fad8ff6c4ce	@serverless-chrome/lambda Started Chrome
2017-07-10 17:20:30.347 (-06:00)	5a96035f-65c6-11e7-9516-0fad8ff6c4ce	@serverless-chrome/lambda It took 543ms to spawn chrome.
2017-07-10 17:20:30.347 (-06:00)	5a96035f-65c6-11e7-9516-0fad8ff6c4ce	Error occured in serverless-plugin-chrome wrapper when trying to ensure Chrome for handler() handler. { chromeFlags: [ '--window-size=1280x1696', '--hide-scrollbars' ] } TypeError: handler.handler is not a function
    at launch.then (/var/task/src/captureScreenshotBinary.js:9:15)
```
Does that make sense to you?

**Comment by adieuadieu** [comment]
Hi Will,

I'm sorry I haven't had a chance to review your updates today. I'll try to get back to you in more detail tomorrow.

My best guess at the error is that somehow the handler is misconfigured in serverless.yml, or there's a bug in the plugin somehow caused when upgrading to the latest version of Serverless.

**Comment by willfarrell** [comment]
@adieuadieu I was really hoping we could drop webpack dep. I kinda have both in the ex folder right now. I'll revisit what I have later in the week, get it tested and finished up. Maybe you can assist with the getting the webpack running smoothly after that.

**Comment by willfarrell** [comment]
Were you able to get it running with some tweaking to other parts of the project?

**Comment by adieuadieu** [comment]
Hi Will — Not yet. Seems to be an issue with the interplay between webpack and the serverless-chrome plugin. Spending some time on it this week.

**Pull request merged**